spec/rspec/rails/matchers/active_job_spec.rb

```diff
@@ -0,0 +1,84 @@
+require "spec_helper"
+require "rspec/rails/matchers/active_job"
+
+RSpec.describe "active_job matchers" do
```

Comment:
Looks like it won't be so easy https://travis-ci.org/rspec/rspec-rails/jobs/83654996#L274
Also, I cannot simply `return` from file, because rspec throws error `/home/wojtek/Projects/rspec-rails/spec/rspec/rails/matchers/active_job_spec.rb: /home/wojtek/Projects/rspec-rails/spec/rspec/rails/matchers/active_job_spec.rb:3: Invalid return (SyntaxError)`
Any other ideas how to pass it?